internal/system/agent/application/executor/util.go

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package executor
+
+import "github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+
+const edgexPrefix = "edgex-"
+
+func addEdgexPrefix(serviceName string) string {
+	switch serviceName {
+	case common.CoreDataServiceKey:
+	case common.CoreCommandServiceKey:
+	case common.CoreMetaDataServiceKey:
+	case common.SupportNotificationsServiceKey:
+	case common.SupportSchedulerServiceKey:
+	case common.SystemManagementAgentServiceKey:
+	case common.SecurityBootstrapperKey:
+	default:
+		return serviceName
```

Comment:
This doesn't solve the inconsistency issue and doesn't work for any device or apps services.

Reply:
Just add the prefix no matter what

Reply:
updated.